beanprice/price.py

```diff
@@ -33,6 +34,17 @@
 import beanprice
 
 
+# A price source.
+#
+#   module: A Python module, the module to be called to create a price source.
+#   symbol: A ticker symbol in the universe of the source.
+#   invert: A boolean, true if we need to invert the currency.
+PriceSource = NamedTuple('PriceSource',
+                         [('module', Any),
+                          ('symbol', str),
+                          ('invert', bool)])
+
```

Comment:
I think that 
```python
class PriceSource(NamedTuple):
    module: Any
    symbol: str
    inveert: bool
```
if immensely more readable.

Reply:
I fully agree, changed.